src/test/java/org/junitpioneer/internal/TestExtensionContext.java

```diff
@@ -42,11 +43,16 @@ public TestExtensionContext(Class<?> testClass, Method testMethod) {
 		this.testMethod = testMethod;
 	}
 
-	// @Override once we baseline against 5.8
+	@Override
 	public ExecutionMode getExecutionMode() {
 		throw NOT_SUPPORTED_IN_TEST_CONTEXT;
 	}
 
+	@Override
```

Comment:
```suggestion
	// @Override once we baseline against 5.9
```
We don't have to update, so we should not.